dirsrvtests/tests/suites/plugins/memberof_test.py

```diff
@@ -2692,6 +2808,12 @@ def test_memberof_auto_add_oc(topology_st):
         11. Success
     """
 
+    memberof = MemberOfPlugin(topology_st.standalone)
+    if (memberof.get_memberofdeferredupdate() and memberof.get_memberofdeferredupdate().lower() == "on"):
+        delay = 3
+    else:
+        delay = 0
```

Comment:
It is repeated so many times that I wonder if we should make an util.py function for that

Reply:
I did the changes for all the CI tests.